java-client/src/test/java11/org/opensearch/client/opensearch/integTest/AbstractGenericClientIT.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.client.opensearch.integTest;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import org.junit.Test;
+import org.opensearch.client.json.JsonpDeserializer;
+import org.opensearch.client.opensearch.core.SearchResponse;
+import org.opensearch.client.opensearch.generic.Bodies;
+import org.opensearch.client.opensearch.generic.Requests;
+import org.opensearch.client.opensearch.generic.Response;
+import org.opensearch.client.opensearch.indices.CreateIndexResponse;
+
+public abstract class AbstractGenericClientIT extends OpenSearchJavaClientTestCase {
+
+    @Test
+    public void shouldReturnSearchResults() throws Exception {
+        final String index = "search_request";
+        createIndex(index);
+
+        try (
+            Response response = javaClient().generic()
+                .execute(
+                    Requests.builder(javaClient())
```

Comment:
`javaClient().builder()` ?

Reply:
Right, I was thinking about that but decided not to:
  - this builder is only for generic client, so definitely not sufficient for sneaking into public API
  - for bodyless requests(), the javaClient() is not needed at all